kairos-prover/default.nix

```diff
@@ -44,6 +45,10 @@
             ./rust-std-Cargo.lock
           ];
         };
+
+        # Proving in CI is disabled because it takes too long.
+        # Proving is a test of risc0, not kairos anyway.
+        RISC0_DEV_MODE = 1;
```

Comment:
**Question:** Does it also affect our Nix-based production setup? Maybe it should be configured inside CI, not in default Nix config.

Reply:
Fixed in https://github.com/cspr-rad/kairos/pull/113.